internal/xds/bootstrap/tlscreds/bundle_ext_test.go

```diff
@@ -224,6 +228,87 @@ func (s) TestCaReloading(t *testing.T) {
 	}
 }
 
+func (s) TestSPIFFEReloading(t *testing.T) {
```

Comment:
Also, please consider writing high level descriptions of the test as docstrings. It really helps folks who are either new and onboarding to the codebase, or are looking into some issue and maybe want to see what tests exists that they can re-purpose for a reproduction of the scenario that they are looking into.

It would be nice for these high level descriptions to contain the following:
- any setup that happens, if it makes sense to document
- what does the test do?
- what does it expect?

Reply:
Done